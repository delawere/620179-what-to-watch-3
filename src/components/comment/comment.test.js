import React from 'react';
import renderer from 'react-test-renderer';
import Comment from './comment.jsx';

it(`Comment renders correctly`, () => {
  const tree = renderer
    .create(
        <Comment date={`12.12.2011`}/>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
