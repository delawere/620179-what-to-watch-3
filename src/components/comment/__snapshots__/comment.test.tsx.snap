// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment renders correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
      style={
        Object {
          "wordBreak": "break-word",
        }
      }
    />
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      />
      <time
        className="review__date"
        dateTime="12.11.2011"
      >
        December 12, 2011
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  />
</div>
`;
